spec/rubocop/config_spec.rb

```diff
@@ -526,6 +544,45 @@
         expect(configuration.cop_enabled?(cop_class)).to be true
       end
     end
+
+    context 'when DisabledByDefault is true' do
+      let(:hash) do
+        {
+          'AllCops' => { 'DisabledByDefault' => true },
+          'Style/Copyright' => { 'Exclude' => ['foo'] }
```

Comment:
Maybe you need `'Enabled' => true` to enable for this cop to be explicitly enabled?